### ISSUE TITLE
Need to use 'fullySpecified' Webpack 5 config

### ISSUE THREAD

**Issue by jschen2** [GoncharukOrg/react-input#30]
I imported this to use in my project that was built using create-react-app and I'm met with the following error when attempting to run it.

> Module not found: Error: Can't resolve 'react/jsx-runtime' in '/node_modules/@react-input/mask/mask/dist/module'
Did you mean 'jsx-runtime.js'?
BREAKING CHANGE: The request 'react/jsx-runtime' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

I'm on react-scripts@5.0.1, webpack@5.91.0, and react@17.0.2. A solution I found was to modify the webpack config by adding
```
     {
        module: {
          rules: [
            {
              test: /\.m?js/,
              resolve: {
                fullySpecified: false
              }
            },
          ],
        },
      }
```
Is there something that can be done that doesn't force me to add this to the config?

**Comment by GoncharukBro** [comment]
A fix for the current issue will be included in `@react-input/mask@2.0.0`.

Important!

Starting with `@react-input/mask@2.0.0`, we removed the `input-mask` event and the `onMask` method, focusing only on using native React events and methods such as `onChange`, since the `input-mask` event cannot be explicitly coordinated with React events and methods, making such usage and event firing order non-obvious.

To use the useful data from the `detail` property of the `input-mask` (`onMask`) event object, you can also use the utilities described in the «[Utils](https://github.com/GoncharukBro/react-input/tree/main/packages/mask#utils)» section.

The release of `@react-input/mask@2.0.0` is expected next week.

**Comment by GoncharukBro** [comment]
Please install the latest version of the package that includes this fix:

```
npm i @react-input/mask@latest
```
